.github/workflows/ci.yml

```diff
@@ -179,7 +179,7 @@ jobs:
       - name: Build test project
         run: forc build --path test-proj
       - name: Run test project's test suite
-        run: cd test-proj && forc test
+        run: cd test-proj && forc test --cargo-test-opts="--features fuel-core-lib"
```

Comment:
Please revert changes to this file, as they're orthogonal to the what this PR targets: "remove fuels-abigen-macro dependency."

Transitively revert any changes that require this file to be changed.

Reply:
In the future, avoid bundling multiple independent issues into a single PR, as the combinatorial explosion in interactions between features and fixes can make it impossible to review quickly.

Reply:
done